internal/lib/lib.go

```diff
@@ -285,16 +285,32 @@ func GetLayer7Only() bool {
 }
 
 var AKOUser string
+var AKOID string
```

Comment:
could you please move this global variable in `akoControlConfig`. Let's try moving all the global variables to the common struct.